packages/roosterjs-editor-dom/lib/edit/adjustInsertPosition.ts

```diff
@@ -281,6 +282,42 @@ function adjustInsertPositionForNotEditableNode(
     return position;
 }
 
+/**
+ * Adjust the position of a table to be one line after another table.
+ */
+function adjustInsertPositionForTable(
+    root: HTMLElement,
+    nodeToInsert: Node,
+    position: NodePosition,
+    range: Range
+): NodePosition {
+    if (
+        (nodeToInsert.childNodes.length == 1 &&
+            getTagOfNode(nodeToInsert.childNodes[0]) == 'TABLE') ||
+        getTagOfNode(nodeToInsert) == 'TABLE'
+    ) {
+        const { element, offset } = position;
+        const previousElementSibling = element.previousElementSibling;
+        const prevElement =
```

Comment:
Suggest use ContentTraverser to find prevous element, which is more accurate than just directly access DOM tree